### PR TITLE
Config activity messages, terminal formatting, !members, !help dynamic command loading

### PR DESCRIPTION
**Pull request by harrego**
1. Added configurable activity messages, you can specify what activity and type of activity the bot is doing in `config.json` under `activityText` and `activityType`

2. I've added pretty formatting to terminal messages with emoji and `chalk` colours

screen shot 2018-08-31 at 10 09 16

3. Added the members command, returns a pretty little embed

4. Added dynamic module loading to the help command, now instead of all commands being hardcoded in a markdown file, the help command will read `modules.json` and return commands using it

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
 const Discord = require('discord.js');
 const Enmap = require('enmap');
+const chalk = require('chalk');
 const Utils = require('./utils.js');
 const fs = require('fs');
 
 let config = require('./config.json');
 let client = new Discord.Client();
 
+var emojiMap = {
```
Comment (Webd01) [inline main.js]: You should use ``const`` instead of ``var``

File: modules/help.js [hunk 2]
```diff
@@ -5,19 +5,20 @@ const fs = require('fs');
 // Usage: !help
 function run(client, message, args) {
   if (args.length == 0) {
+
+    var modules = require("./modules.json");
```
Comment (Webd01) [inline modules/help.js]: Use let

File: modules/members.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const Discord = require('discord.js');
+
+// Returns the member count of the server.
+// Usage: !members
+function run(client, message, args) {
+  var embed = client.utils.createEmbed(
```
Comment (Webd01) [inline modules/members.js]: You should change var for const here too :)

**Comment by harrego** [comment]
@Webd01 thanks! I just fixed these in 6ce06d0

**Pull request merged**